languages/go/xorm/migrations.go

```diff
@@ -3,29 +3,8 @@ package main
 import (
 	"database/sql"
 	"log"
-	"os"
 )
 
-func InstallEql(engine *sql.DB) {
-	log.Println("Start installing custom types and function")
-	installDsl(engine)
-
-}
-
-// Installing EQL
-func installDsl(engine *sql.DB) {
-	path := "../../../release/cipherstash-encrypt-dsl.sql"
```

Comment:
removed as this is now being installed on init of docker with a download directly from eql